src/style.css

```
@@ -1,3 +1,61 @@
+html {
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding:0 50px;
+}
+
+.logo_link {
+  width: 40px;
+  height: 40px;
+  }
+
+.list__items {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.list__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  }
+
+.nav__link:hover,
+.is-active {
+  color: #00ACDC;
 }
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00ACDC;
+  }
+
+
```

```suggestion
  background-color: #00ACDC;
}

```
Fix formatting in all places, and remove second blank line, it should be only one